templates/theme-footer.html

```diff
@@ -1,10 +1,13 @@
 ## mako
-<%! from django.core.urlresolvers import reverse %>
+<%!
+  from django.core.urlresolvers import reverse
+  from django.utils.translation import ugettext as _
+%>
 <%namespace name='static' file='/static_content.html'/>
-<%! from datetime import datetime %>
-<%! from pytz import UTC %>
-<%! from django.utils.translation import ugettext as _ %>
-
+<%!
+  from datetime import date
+%>
+<!--
```

Comment:
Generally it's considered a bad practice to leave blocks of commented-out code, because it raises questions and puzzles future developer. If this is going to be a permanent change, just delete the code. If you're fiddling with the code -- it's better to do it in a separate branch or fork.